katran/lib/KatranLb.cpp

```diff
@@ -424,11 +424,13 @@ bool KatranLb::changeMac(const std::vector<uint8_t> newMac) {
     ctlValues_[kMacAddrPos].mac[i] = newMac[i];
   }
   if (!config_.testing) {
+    lbStats_.bpfCalls++;
```

Comment:
On a lot of these places, I think it's better if we increment bpfCalls counter only if those calls succeed (and increment failedCall counter if they fail like you do). 
Or at least rename this variable to bpfCallAttempts?

Reply:
i was interested in overall number of syscalls. but yeah, aside from my curiosity i dont thinks there are lot of value in it. agree, we can drop it for now and add in future if we would have more solid usecase